guides/_subsections/add-and-edit-recording-metadata-labels.md

```diff
@@ -10,72 +10,72 @@ Users can attach metadata or custom labels to JDK flight recordings managed by C
+        When creating a custom <code>Flight Recording</code> with <b>Cryostat</b>, expand the form section <i>Show metadata options</i>. Click <i>Add Label</i> and add <i>key-Value</i> <code>Label</code> pairs to the form. Finally, click <i>Create</i> to attach the <code>Labels</code> and create the <code>Recording</code>.
       "
     %}
   </li>
   <li>
     {% include howto_step.html
-      summary="View your labels on the Active Recordings Table"
+      summary="View your <code>Labels</code> on the <i>Active Recordings Table</i>"
       image-name="2.3.0/add-and-edit-recording-metadata-labels-2.png"
       text="
-        The new recording will appear in the Recordings tab with your custom label as well as default labels containing information about the selected recording template.
+        The new <code>Recording</code> will appear in the <i>Recordings Tab</i> with your <code>Custom Label</code> as well as default <code>Labels</code> containing information about the selected <code>Recording</code> template.
       "
     %}
   </li>
   <li>
     {% capture edit-an-existing-label-text %}
     <p>
-      Recording labels can also be edited after recordings have been created or re-uploaded to archives. It looks like the custom label in our example contains a typo - we can fix the typo by editing the label. First select a recording from the table with the 🗹 checkbox. Then, click the <i>Edit Labels</i> button to bring up the label drawer. Finally, click the <i>Edit</i> button that appears from the drawer.
+      <code>Recording Labels</code> can also be edited after <code>Recordings</code> have been created or re-uploaded to archives. It looks like the <code>Custom Label</code> in our example contains a typo - we can fix the typo by editing the <code>Label</code>. First select a <code>Recording</code> from the table with the 🗹 checkbox. Then, click the <i>Edit Labels</i> button to bring up the label drawer. Finally, click the <i>Edit</i> button that appears from the drawer.
       <br><br>
       <a href="{{ site.url }}/images/2.3.0/add-and-edit-recording-metadata-labels-4.png" target="_blank">
         <img src="{{ site.url }}/images/2.3.0/add-and-edit-recording-metadata-labels-4.png">
       </a>
       <br><br>
-      The labels section will appear as a form where you can add, edit, or delete existing labels, just like before when we created the recording. Fix the typo, and click <i>Save</i> to save your edited labels.
+      The <code>Labels</code> section will appear as a form where you can <i>Add</i>, <i>Edit</i>, or <i>Delete</i> existing <code>Labels</code>, just like before when we created the <code>Recording</code>. Fix the typo, and click <i>Save</i> to save your edited <code>Labels</code>.
```

Comment:
... just like when we create the <code>Recording</code>. I think having the "before" is redundant and too wordy.